### PR TITLE
2020 methodology updates

### LINKED ISSUE #1404: Add more content to 2020 Methodology page
Following on from #1383 we merged this without a full review so raising separate issue for this.

- [ ] - @rviscomi talked about adding content on CSS and Rework AST
- [ ] - Number of queries is currently set to TBC

### PR DESCRIPTION
**Pull request by rviscomi**
Fixes #1404 

@LeaVerou please review the section on Rework Utils and feel free to suggest anything else you'd like to say about the tools used by the CSS chapter.

@bazzadp sorry for all the small changes, I expect this will throw off the ongoing translation efforts, but I think these are good improvements. I needed to add a section about inclusion to the planning section (which I renamed from "Brainstorming", the difference from 2019 is ok)

### CODE REVIEW

File: src/templates/en/2020/methodology.html
Comment (bazzadp) [inline src/templates/en/2020/methodology.html:420]: Not sure we need the "finally" here? Reads like part of a list when it's not.

```suggestion
        In July 2020, after rounds of brainstorming and nominations, 22 chapters were solidified and we formed content teams for each chapter tasked with writing, reviewing, and analysis.
```

File: src/templates/en/2020/methodology.html
Comment (bazzadp) [inline src/templates/en/2020/methodology.html:428]: We added a LOT of custom metrics this year. Should we expand this out to acknowledge that effort? And perhaps the longer term gains of that?

File: src/templates/en/2020/methodology.html
Comment (bazzadp) [inline src/templates/en/2020/methodology.html:108]: Should we update this example SQL by picking one from the 2020 Almanac?

**Review by bazzadp — COMMENTED** [review]
Yes that's a lot more change than I was expecting to be honest! But it's the right thing to do so we'll deal with it for the translations. That was my fault trying to get ahead of the game even though I knew this was liable to change.

So French 2020 Methodology is already merged. Will try to get that updated ASAP after this is merged.

Japanese is ready to go (without these changes) in #1421 . I'd still be tempted to merge that and iterate. What's your thoughts? Alternatively can hold off merging that (switch to draft maybe) and insist these changes (once approved and merged) are incorporated into that branch?

Chinese (Simplified) is not ready to go (#1422 ) and maybe we just cancel that PR and do it from scratch. Or switch to draft, assign to Chinese team and insist they incorporate these changes before merging. Again what's your thoughts?

On the one hand there's a lot of niggly changes here, which could be easily missed if attempting to start from 2019, on the other hand the vast majority isn't changing so starting from 2019 like I did gets the bulk of it done.

File: src/templates/en/2020/methodology.html
Comment (rviscomi) [reply src/templates/en/2020/methodology.html:108]: +1 when more queries are merged it'd be nice to update this to 2020.

File: src/templates/en/2020/methodology.html
Comment (rviscomi) [reply src/templates/en/2020/methodology.html:428]: +1 I'll think of something to say on that

File: src/templates/en/2020/methodology.html
Comment (rviscomi) [reply src/templates/en/2020/methodology.html:420]: I felt like this needed a transition word to move from the inclusion process to team formation.

**Comment by rviscomi** [comment]
Regarding your questions about how to proceed with translations already in flight, I agree it's ok to merge the translation work that's already been completed and iterate as needed.

File: src/templates/en/2020/methodology.html
Comment (rviscomi) [reply src/templates/en/2020/methodology.html:108]: Resolving this for now but we can revisit before launch.

**Comment by LeaVerou** [comment]
I definitely have things to add here. What's the deadline?

**Review by bazzadp — APPROVED** [review]

**Comment by rviscomi** [comment]
@LeaVerou we're planning to launch the early access version of the Methodology today, but we can make iterative changes to it up until the full launch on December 9. Any objections to publishing the version in this PR today and adding more detail as we get closer to launch?

**Comment by LeaVerou** [comment]
Cool, I just read the paragraph and I'm fine with it!
I just pushed a small change to separate Parsel and Rework Utils, since the former is an entirely separate micro library.
Once we launch the CSS chapter we should also add a link to the playground, so that people can run the metrics against their own CSS, but that's not for now.

**Comment by rviscomi** [comment]
Great, thanks! Two small updates to your change so that the CSS chapter and your name are links to the chapter/contributor pages. This is ready to merge/launch 🚀

**Pull request merged**
